Begun creating the UI

abacall opened the pull request:
Initial opinions or suggestions?

abacall committed:
initial ui stuff

abacall committed:
Begun creating the UI for the Election Selection Page

AlexFox97 commented:
Looks like a good start however please alter line 38 of HelloWorld.java so that the file location is not specific to one computer. Also HelloWorld.java could do with a more meaningful name.

abacall committed:
Added the other pages as copies of the inital Election Selection Page.

Renamed select_election.fxml to Page_Vote_Selection.fxml for readability
purposes.

AlexFox97 reviewed (approved):
XML changes and additions look good
